internal/command/action.go

```diff
@@ -67,6 +74,15 @@ func runAction(ac *ActionConfig) error {
 		err     error
 	)
 
+	if ac.ShowProviderVersions{
+		output, err := printProviderVersions()
+		if err != nil {
+			return err
+		}
+		fmt.Print(output)
```

Comment:
nit: One newline above here - the error checks usually are separated from code right after it.